pkg/pb/orders.pb.go

```diff
@@ -3,17 +3,15 @@
 
 package pb
 
-import proto "github.com/gogo/protobuf/proto"
-import fmt "fmt"
-import math "math"
-import _ "github.com/gogo/protobuf/gogoproto"
-import timestamp "github.com/golang/protobuf/ptypes/timestamp"
-
-import time "time"
-
 import (
-	context "golang.org/x/net/context"
+	context "context"
```

Comment:
Why did this file change? Which version of protoc / gogo are you using or did you manually format the files? (These may have been generated with a different version previously as well.)

Reply:
I just use `pb.gen.go`

Reply:
Yeah, it looks like somebody used older protobuf / protoc or some other version.

Reply:
Yeah, it looks like somebody used older protobuf / protoc or some other version.